### PR TITLE
Added  eslint curly error config and fixed errors

### LINKED ISSUE #620: [DevEx]: Apply ESLint 'curly' rule to require curly braces around all block statements
### Improvement Summary

Inconsistent use of `{}` can make code harder to read. We should make this more consistent by applying a lint rule disallowing single-line block statements.

### Detailed Description

We can accomplish this with the 'all' setting of the [eslint curly](https://eslint.org/docs/latest/rules/curly) rule. We should configure it's error level to 'error' so that the rule is mandatory. This will also require fixing all the areas that violate the rule.

### PR DESCRIPTION
**Pull request by Mohanth**
Fix #620

### CODE REVIEW

File: api/controllers/game/face-card.js [hunk 1]
```diff
@@ -65,7 +65,7 @@ module.exports = function (req, res) {
         victory,
       });
       // If the game is over, clean it up
-      if (victory.gameOver) await gameService.clearGame({ userId: req.session.usr });
+      if (victory.gameOver) {await gameService.clearGame({ userId: req.session.usr });}
```
Comment (seriouslysean) [inline api/controllers/game/face-card.js:68]: I think these should always be on a new line.

Comment (Mohanth) [reply api/controllers/game/face-card.js]: Modified the same in all files and updated.

**Review by seriouslysean — APPROVED** [review]
Nice, this is great! I love a consistent codebase! Thank you so much for your contribution, leaving final approval to @itsalaidbacklife.

File: api/controllers/game/untargeted-one-off.js [hunk 2]
```diff
@@ -26,22 +26,28 @@ module.exports = function (req, res) {
                 switch (card.rank) {
                   case 3:
                     if (game.scrap.length < 1)
-                      return Promise.reject({
+                      {
+                        return Promise.reject({
                         message: 'You can only play a 3 as a one-off, if there are cards in the scrap pile',
```
Comment (itsalaidbacklife) [inline api/controllers/game/untargeted-one-off.js:31]: Here too

File: api/controllers/game/untargeted-one-off.js [hunk 3]
```diff
@@ -26,22 +26,28 @@ module.exports = function (req, res) {
...
+                        });
+                      }
                     break;
                   case 5:
                   case 7:
                     if (!game.topCard)
-                      return Promise.reject({
+                      {
+                        return Promise.reject({
                         message: "You can't play that card as a one-off, unless there are cards in the deck",
```
Comment (itsalaidbacklife) [inline api/controllers/game/untargeted-one-off.js:48]: And here

**Review by itsalaidbacklife — COMMENTED** [review]
Nice work. Thank you for your contribution! I have one nitpick about the indentation of error messages that applies in a few different places. With those addressed this will be good to go

File: api/controllers/game/resolve.js [hunk 4]
```diff
@@ -281,9 +281,11 @@ module.exports = function (req, res) {
               case 'point': {
                 const targetCard = opPoints.find((point) => point.id === game.oneOffTarget.id);
                 if (!targetCard)
-                  return Promise.reject({
+                  {
+                    return Promise.reject({
                     message: `Could not find target point card ${game.oneOffTarget.id} to return to opponent's hand`,
```
[the remark this answers is not in the document]
Comment (seriouslysean) [reply api/controllers/game/resolve.js:286]: I actually have a question about this. I assumed the fix lint command was run, would it make sense to double check the rule and ensure it's formatting as expected?

[the remark this answers is not in the document]
Comment (Mohanth) [reply api/controllers/game/resolve.js:286]: eslint --fix is not fixing the indentations. Maybe there is a conflict with .prettier and eslintrc configurations

[the remark this answers is not in the document]
Comment (Mohanth) [reply api/controllers/game/resolve.js]: Pushed changes by adding required indentation for the files

**Comment by Mohanth** [comment]
Requesting for review and approval.

**Review by itsalaidbacklife — COMMENTED** [review]
Thank you @Mohanth this looks good. We just merged a restructuring PR that caused some merge conflicts but they should be straightforward to fix since it was just about moving files around.

Essentially we've combined views and components into a new `src/routes` folder with one subfolder per page that includes both the Page/View and a subfolder with all the components for the page. So:
1. `src/componentGameView/MoveChoiceOverlay.vue` has been moved to `src/routes/game/components/MoveChoiceOverlay.vue`
2. `src/componentGameView/SevenDoubleJacksDialog.vue` has been moved to `src/routes/game/components/SevenDoubleJacksDialog.vue`
3. `src/views/GameView.vue` has been moved to `src/routes/game/GameView.vue`
4. `src/views/HomeView.vue` has been moved to `src/routes/home/HomeView.vue`

**Comment by Mohanth** [comment]
Conflicts resolved.

**Review by itsalaidbacklife — APPROVED** [review]
Great work, this helps to standardize the codebase. Thanks for your contribution!

**Pull request merged**